src/common/public/oscap.h

```diff
@@ -91,7 +91,8 @@ typedef enum oscap_document_type {
 	OSCAP_DOCUMENT_XCCDF,                 ///< XCCDF benchmark file
 	OSCAP_DOCUMENT_CPE_LANGUAGE,          ///< CPE language file
 	OSCAP_DOCUMENT_CPE_DICTIONARY,        ///< CPE dictionary file
-	OSCAP_DOCUMENT_CVE_FEED,	      ///< CVE NVD feed
+	OSCAP_DOCUMENT_CVE_FEED,	          ///< CVE NVD feed
+	OSCAP_DOCUMENT_CVRF_FEED,	          ///< CVRF feed
```

Comment:
the whitespace after the enum is mixed tabs and spaces, please only use spaces there